package.json

```diff
@@ -1,28 +1,61 @@
 {
   "name": "streamr-client",
-  "version": "0.9.3",
-  "description": "JS client for subscribing to Streamr streams",
+  "version": "0.10.0",
+  "description": "JavaScript client library for Streamr",
   "repository": {
     "type": "git",
     "url": "git://github.com/streamr-dev/streamr-client.git"
   },
-  "main": "streamr-client.js",
+  "main": "dist/streamr-client.node.js",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "test": "mocha test"
+    "build": "webpack --env dev --target web && webpack --env dev --target node && webpack --env prod --target web && npm run test",
```

Comment:
No script to build for node?

Reply:
What do you mean? The `build` script does build for node.